platform-sdk/swirlds-state-api/src/test/java/com/swirlds/state/spi/EmptyWritableStatesTest.java

```diff
@@ -26,31 +27,24 @@ void empty() {
     @Test
     @DisplayName("Contains is always false")
     void contains() {
-        assertThat(states.contains(FRUIT_STATE_KEY)).isFalse();
-    }
-
-    @Test
-    @DisplayName("Throws NPE if the key is null")
-    void nullKey() {
-        //noinspection DataFlowIssue
-        assertThatThrownBy(() -> states.get(null)).isInstanceOf(NullPointerException.class);
+        assertThat(states.contains(FRUIT_STATE_ID)).isFalse();
     }
 
     @Test
     @DisplayName("Throws IAE for any non-null key")
```

Comment:
Seems like display names for 3 tests below should be updated:
```suggestion
    @DisplayName("Throws IAE for any unknown key")
```